src/libcore/num/wrapping.rs

```diff
@@ -26,9 +32,16 @@ use intrinsics::{i16_mul_with_overflow, u16_mul_with_overflow};
 use intrinsics::{i32_mul_with_overflow, u32_mul_with_overflow};
 use intrinsics::{i64_mul_with_overflow, u64_mul_with_overflow};
 
+#[unstable(feature = "core", reason = "may be removed, renamed, or relocated")]
+#[deprecated(since = "1.0.0", reason = "moved to inherent methods")]
 pub trait WrappingOps {
+    // Methods stable to avoid warning on use via prelude
```

Comment:
Yes. I can try removing these attributes again, it may have been resolved by changes elsewhere.